pandas/tests/indexing/test_loc.py

```diff
@@ -1130,3 +1131,17 @@ def test_loc_with_period_index_indexer():
     tm.assert_frame_equal(df, df.loc[list(idx)])
     tm.assert_frame_equal(df.iloc[0:5], df.loc[idx[0:5]])
     tm.assert_frame_equal(df, df.loc[list(idx)])
+
+
```

Comment:
can you put in pandas/tests/indexing/test_chaining_and_caching.py instead with the rest of these tests

Reply:
sure

Reply:
sure

Reply:
There is already a function that test if chained assignment raise a warning `SettingWithCopyWarning` in the case of warn option. We need just modify that function to detect also `SettingWithCopyError` in the case of raise option.